.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 AllCops:
   DisplayCopNames: true
-  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.3
```

Comment:
I think we can just remove this - it is inferred from `.ruby-version` https://github.com/bbatsov/rubocop/blob/7a2ade6c980f2df04b4625ba07d50d65b69a7465/manual/configuration.md#setting-the-target-ruby-version